images/linux/scripts/installers/microsoft-edge.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+################################################################################
+##  File:  microsoft-edge.sh
+##  Desc:  Installs Microsoft Edge
+################################################################################
+
+REPO_URL="https://packages.microsoft.com/repos/edge"
+
+wget -qO- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > /usr/share/keyrings/microsoft-edge.gpg
+# Specify an arch as Microsoft repository supports armhf and arm64 as well
+echo "deb [arch=amd64 signed-by=/usr/share/keyrings/microsoft-edge.gpg] $REPO_URL stable main" > /etc/apt/sources.list.d/microsoft-edge.list
+
+apt-get update
+apt-get install --no-install-recommends microsoft-edge-stable
+
+rm /usr/share/keyrings/microsoft-edge.gpg
```

Comment:
use variables for the /usr/share/keyrings/microsoft-edge.gpg and /etc/apt/sources.list.d/microsoft-edge.list